lib/inspec/plugin/v2.rb

```diff
@@ -27,4 +37,8 @@ def self.plugin(version, plugin_type = nil)
     return Inspec::Plugin::V2::PluginBase if plugin_type.nil?
     Inspec::Plugin::V2::PluginBase.base_class_for_type(plugin_type)
   end
+
+  def self.config_dir
```

Comment:
We can remove this since it is here now right?

https://github.com/inspec/inspec/blob/master/lib/inspec/globals.rb#L2